### DISCUSSION TITLE
%E7%A0%94%E5%AD%A6%E7%AF%87/%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F%E4%B8%8E%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F

### DISCUSSION THREAD

**Discussion by giscus** [SurviveXJTU/SurviveXJTU.github.io#6] [Announcements]
# %E7%A0%94%E5%AD%A6%E7%AF%87/%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F%E4%B8%8E%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F

西安交大生存指南，更适合西交大学子的生存指南。

https://survivexjtu.github.io/%E7%A0%94%E5%AD%A6%E7%AF%87/%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F%E4%B8%8E%E5%9B%9B%E7%A7%8D%E5%AD%A6%E7%94%9F.html

**Comment by Layoomiety-why** [comment]
强基也是有淘汰机制的(T⌓T)
